aiohttp/streams.py

```diff
@@ -490,7 +492,10 @@ def _read_nowait(self, n: int) -> bytes:
         return b"".join(chunks) if chunks else b""
 
 
-class EmptyStreamReader(AsyncStreamReaderMixin):
+class EmptyStreamReader(StreamReader):
```

Comment:
Add this
```suggestion
class EmptyStreamReader(StreamReader):  # lgtm [py/missing-call-to-init]
```
to ignore https://lgtm.com/projects/g/aio-libs/aiohttp/snapshot/0e21998af5e6fc75741c4f9e1c43696c349dd279/files/aiohttp/streams.py?sort=name&dir=ASC&mode=heatmap#x797aa799000fbf74:1

P.S. LGTM will take the comment into account only upon merge.